zaza/openstack/charm_tests/mysql/test_prometheus_mysql_exporter.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2022 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""MySQL Prometheus Exporter Testing."""
+
+import json
+
+import zaza.model as zaza_model
+from zaza.openstack.charm_tests.mysql.tests import MySQLBaseTest
+
+
+class PrometheusMySQLExporterTest(MySQLBaseTest):
+    """Functional tests check prometheus exporter."""
+
+    @classmethod
+    def setUpClass(cls, application_name=None):
+        """Run class setup for running mysql tests."""
+        super().setUpClass(application_name="mysql-innodb-cluster")
+        cls.application = "mysql-innodb-cluster"
+
+    def _exporter_http_check(
+        self,
+        cmd,
+        expected,
+    ):
+        """Exec check cmd on each unit in the application.
+
+        :param str cmd: The check command run on unit.
+        :param str expected: Expected result code.
```

Comment:
Although we don't strictly enforce params in docstrings on zaza-opnestack-tests, if you do include a `:param:` it would be nice to also include the `:type:` for the param, please.

Reply:
Updated